doc/cephfs/quota.rst

```diff
@@ -28,7 +28,8 @@ Limitations
    Linux kernel clients >= 4.17 support CephFS quotas but only on
    mimic+ clusters.  Kernel clients (even recent versions) will fail
    to handle quotas on older clusters, even if they may be able to set
-   the quotas extended attributes.
+   the quotas extended attributes. Quota resize is supported only on
+   nautilus+ clusters.
```

Comment:
Quota resize has been supported since Mimic. Maybe you're thinking of the volumes plugin quota resize?